core/templates/pages/topics-and-skills-dashboard-page/modals/unassign-skill-from-topics-modal.component.html

```diff
@@ -4,14 +4,26 @@ <h3>
   </h3>
 </div>
 <div class="modal-body">
-  <span>This skill is assigned to these following topics, Tick the topics that you want this skill to be unassigned from.</span>
+  <div><em>The topics with this skill as their only diagnostic test skill are not eligible for the unassignment.</em></div><br>
+  <span *ngIf="topicsAssignmentsAreFetched && eligibleTopicsForUnassignment.length !== 0">This skill is assigned to these following topics. Tick the topics that you want this skill to be unassigned from.</span>
+  <div>
+    <span *ngIf="topicsAssignmentsAreFetched && eligibleTopicsForUnassignment.length === 0">No eligible topics for unassignment.</span>
```

Comment:
Let's have this be more conversational: "This skill is not currently assigned to any topics."

Reply:
Updated